src/main/java/com/zzpj/backend/api/v1/PurchaseController.java

```diff
@@ -22,19 +27,27 @@ public PurchaseController(PurchaseServiceLocal purchaseService) {
     }
 
     @GetMapping
-    public ResponseEntity<List<Purchase>> get() {
-        return new ResponseEntity<>(purchaseService.getAllPurchases(), HttpStatus.OK);
+    public ResponseEntity<List<GetAllPurchaseDTO>> getAll() {
+        return new ResponseEntity<>(PurchaseGetAllMapper.getAllPurchaseDTO(purchaseService.getAllPurchases()), HttpStatus.OK);
+    }
+
+    //TODO prawdopodobnie login będzie wyciągany w inny sposób
```

Comment:
Please never add comments to the code version that you would like to be merged to develop. Next time mark this branch as WIP and then after you finish everything request review.